### PR TITLE
[Refactor]  Remove the need for mocking the user session with OCMock (Part 3)

### PR DESCRIPTION
**Pull request by typfel**
## What's new in this PR?

In order to convert the ZMUserSession to Swift we need to remove any usage of OCMock where it mocks the user session.

---

Following the strategy we've arrived at in https://github.com/wireapp/wire-ios-sync-engine/pull/1164 I'm removing more usage of the `mockTransportSession`.

### CODE REVIEW

File: Tests/Source/DatabaseTest.swift [hunk 1]
```diff
@@ -77,19 +77,54 @@ class DatabaseTest: ZMTBaseTest {
...
+//        [self.syncMOC performGroupedBlockAndWait:^{
+//            self.syncMOC.userInfo[@"TestName"] = self.name;
+//            [self.syncMOC addGroup:self.dispatchGroup];
+//            [self.syncMOC saveOrRollback];
+//
+//            [self.syncMOC setZm_userInterfaceContext:self.uiMOC];
+//            [self.syncMOC setPersistentStoreMetadata:@(notificationContentVisible) forKey:@"ZMShouldNotificationContentKey"];
+//            self.syncMOC.zm_fileAssetCache = fileAssetCache;
+//            self.syncMOC.zm_userImageCache = userImageCache;
+//            }];
```
Comment (johnxnguyen) [inline Tests/Source/DatabaseTest.swift]: What's this?

Comment (billypchan) [inline Tests/Source/DatabaseTest.swift]: old objc code?

Comment (typfel) [reply Tests/Source/DatabaseTest.swift]: I ported some code from `MessagingTests.m` which should be deleted.

**Comment by typfel** [comment]
@zenkins release patch

**Pull request merged**